vuepress/docs/next/docs/ecr/ecr-bundle-details.md

```diff
@@ -99,62 +105,51 @@ Here is an example of a plugin descriptor:
+Each plugin is deployed into Kubernetes using composite names. The first string is created by reading the descriptor file. The second string is appended autonomously by Kubernetes and 31 characters long. Each Kubernetes pod name length must not exceed 63 characters or the deployment will fail.
 
-In the previous example of the descriptor a possible resulting pod name will be this one in case the `deploymentBaseName` property is not present:
+**DeploymentBaseName**
 
-`entando-my-image-1-0-0-server-deployment-6f86f459wj9k`
+Initially, the first segment of the pod name was generated by concatenating and manipulating the `image` field value. However, this approach could result in a pod name longer than 63 characters.
+To solve this problem, another (optional) property is available: `deploymentBaseName`, which accepts a string of up to 32 characters. If present, its value will be used as the first part of the pod name instead of the variant of the `image` value.
 
-and this one if the `deploymentBaseName` property is present:
+If the `deploymentBaseName` property is not present in the previous descriptor example, a possible pod name is `entando-my-image-1-0-0-server-deployment-6f86f459wj9k`. If the `deploymentBaseName` property is present, a possible pod name is `myplugin-server-deployment-6f86f459wj9k`.
 
-`myplugin-server-deployment-6f86f459wj9k`
-
-Please note that if you are using the `deploymentBaseName` property and you want to install more versions of the same plugin at the same time, you need to specify different values for `deploymentBaseName`, maybe including the plugin version.
+If you are using the `deploymentBaseName` property and want to install more versions of the same plugin at the same time, you need to specify different values for `deploymentBaseName` (perhaps including the plugin version).
 
 ### Permissions
 
-The `permissions` property specifies a list of coupled clientId and roles that will be bound in Keycloak.
-
-To find them you can open the Keycloak console and then navigate to _clients_ → _awesomeplugin-server_ → _Service Account Roles_.
+The `permissions` property specifies a list of coupled clientIds and roles that will be bound in Keycloak. To find them, open the Keycloak console and navigate to _clients_ → _awesomeplugin-server_ → _Service Account Roles_.
 
-Currently using a non-existing clientId or role will be ignored and the plugin deployment will succeed without errors.
+Currently, use of a nonexistent clientId or role will be ignored and will not cause errors in the plugin deployment.
 
 
 ::: tip
-The more verbose CRD plugin descriptor format is deprecated as of Entando 6.3 but is documented [here](../../../v6.2/docs/ecr/ecr-bundle-details.md). 
+The verbose CRD plugin descriptor format is deprecated as of Entando 6.3, but is documented [here](../../../v6.2/docs/ecr/ecr-bundle-details.md). 
```

Comment:
@nshaw, what is a "verbose" CRD plugin descriptor format? I looked it up and nothing came up...

Reply:
@jyunmitch , it's not really a thing. At the time we had 2 formats that a plugin descriptor could use. The "verbose" version was close to a full Custom Resource YAML description but we stopped supporting it after 6.3.0 so we should remove this sentence. I made a comment about that but I can't see it in this view...

Reply:
@jyunmitch the yaml files are different

Reply:
@nshaw i didn't see that comment, either. will remove